src/main/java/com/rarchives/ripme/ripper/VideoRipper.java

```diff
@@ -109,33 +117,33 @@ public void downloadCompleted(URL url, File saveAs) {
             logger.error("Exception while updating observer: ", e);
         }
     }
+
     @Override
     public void downloadErrored(URL url, String reason) {
-        if (observer == null) {
+        if (observer == null)
             return;
-        }
+
         observer.update(this, new RipStatusMessage(STATUS.DOWNLOAD_ERRORED, url + " : " + reason));
         checkIfComplete();
     }
+
     @Override
     public void downloadExists(URL url, File file) {
-        if (observer == null) {
+        if (observer == null)
             return;
-        }
+
         observer.update(this, new RipStatusMessage(STATUS.DOWNLOAD_WARN, url + " already saved as " + file));
         checkIfComplete();
     }
 
     @Override
     public String getStatusText() {
-        StringBuilder sb = new StringBuilder();
-        sb.append(getCompletionPercentage())
-          .append("% ")
-          .append(" - ")
-          .append(Utils.bytesToHumanReadable(bytesCompleted))
-          .append(" / ")
-          .append(Utils.bytesToHumanReadable(bytesTotal));
-        return sb.toString();
+        return String.valueOf(getCompletionPercentage()) +
```

Comment:
I acknowledge there may actually be no difference because of compiler optimizations, but until I have a chance to look into it, I don't want to make this call just yet. https://stackoverflow.com/questions/1532461/stringbuilder-vs-string-concatenation-in-tostring-in-java